Q_hub/static/style.css

```diff
@@ -0,0 +1,83 @@
+nav{
```

Comment:
yeah go according to the issue. It says` django initial setup`. So it would be good not to include frontend related files in this PR. For frontend related files open up a different issue & then create a PR. Let me know if you have queries.